repos/system_upgrade/el7toel8/actors/unsupportedupgradecheck/actor.py

```diff
@@ -0,0 +1,62 @@
+from leapp import reporting
+from leapp.actors import Actor
+from leapp.models import Report
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+
+class UnsupportedUpgradeCheck(Actor):
+    """
+    Checks enviroment variables and produces a warning report if the upgrade is unsupported.
+
+    Upgrade is unsupported if any LEAPP_DEVEL_* variable is used or an experimental actor is enabled.
+    This can be overridden by setting the variable LEAPP_UNSUPPORTED (at user's own risk).
+    """
+
+    name = 'unsupported_upgrade_check'
+    consumes = ()
+    produces = (Report,)
+    tags = (ChecksPhaseTag, IPUWorkflowTag)
+
+    def process(self):
+        conf = self.configuration
```

Comment:
Nit: conf is not used anywhere else, I'd abandon it and have
`leapp_vars = self.configuration.leapp_env_vars`

Reply:
Alright. :)